manual_flagging/flagging.py

```diff
@@ -315,18 +315,21 @@ def create_bins_and_labels(input_list):
     # Make a copy of the data frame to edit
     print(f"\n Enacting group threshold and creating ptax data for {df_name}")
     df_copy = df_info["df"].copy()
-    df_copy = flg.group_size_adjustment(
-        df=df_copy,
-        stat_groups=df_info["columns"],
-        min_threshold=inputs["min_groups_threshold"],
-        condos=df_info["condos_boolean"],
-    )
+
     df_copy = flg.ptax_adjustment(
         df=df_copy,
         groups=df_info["columns"],
         ptax_sd=inputs["ptax_sd"],
         condos=df_info["condos_boolean"],
     )
+
+    df_copy = flg.group_size_adjustment(
+        df=df_copy,
+        stat_groups=df_info["columns"],
+        min_threshold=inputs["min_groups_threshold"],
+        condos=df_info["condos_boolean"],
+    )
+
     """
```

Comment:
This is the second piece of problematic code with regards to the false positives. The group size adjustment seems to have been happening before the ptax outliers were assigned. The actual assignment of `sv_is_outlier` to `1` happened in `finish_flags()`, which also ran after the `group_size adjustment` The fix in this PR puts all of the ptax handling directly in the `ptax_adjustment()` function and swaps the order of the `ptax_adjustment` and `group_size_adjustment()`.